pkg/daemon/ceph/osd/volume.go

```diff
@@ -178,6 +182,13 @@ func (a *OsdAgent) configureCVDevices(context *clusterd.Context, devices *Device
 		}
 		osds = append(osds, lvmOsds...)
 
+		// List existing OSD(s) configured with ceph-volume raw mode
+		rawOsds, err = GetCephVolumeRawOSDs(context, a.clusterInfo, a.clusterInfo.FSID, block, "", "", false)
```

Comment:
For OSDs on PVCs will this duplicate the OSD in the list? Or do OSDs on PVCs return separately? I don't recall.

Reply:
It must not be duplicated since the testing looks good.